src/Parser/Monk/Windwalker/CHANGELOG.js

```diff
@@ -1,4 +1,5 @@
 export default `
+17-10-2017 - File structure updates for module. Should be no impact to end user (by anomoly)
```

Comment:
I thought the CHANGELOG files were supposed to be specifically for the benefit of the end user. This shouldn't have a changelog entry.